### PR TITLE
octeon: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
Splits up DEVICE_TITLE into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT.

### CODE REVIEW

File: target/linux/octeon/image/Makefile [hunk 1]
```diff
@@ -23,22 +23,24 @@ define Build/strip-kernel
 endef
 
 define Device/generic
+  DEVICE_MODEL := Generic
```
Comment (mwarning) [inline target/linux/octeon/image/Makefile]: Should there be a vendor set?

Comment (dangowrt) [reply target/linux/octeon/image/Makefile]: Just keep DEVICE_TITLES for Generic builds which don't belong to any specific board.

**Review by dangowrt — CHANGES_REQUESTED** [review]

File: target/linux/octeon/image/Makefile [hunk 2]
```diff
@@ -23,22 +23,24 @@ define Build/strip-kernel
 endef
 
 define Device/generic
-  FILESYSTEMS := ext4
   DEVICE_TITLE := Generic
+  FILESYSTEMS := ext4
```
Comment (adrianschmutzler) [inline target/linux/octeon/image/Makefile]: Why is this showing up as diff here?

Comment (mwarning) [reply target/linux/octeon/image/Makefile]: I will fix it.

Comment (adrianschmutzler) [reply target/linux/octeon/image/Makefile]: Just realized that my question might sound rude from a different angle, that was not my intention.
Thanks for fixing.

**Review by adschm — APPROVED** [review]
LGTM.

File: target/linux/octeon/image/Makefile (hunk 2 above)
Comment (mwarning) [reply target/linux/octeon/image/Makefile]: It did not sound rude. Maybe I made the impression? All fine. Let's keep hacking. :-)

**Comment by lynxis** [comment]
it has been merged.

**Pull request closed without merging**